main.js

```diff
@@ -1,5 +1,29 @@
-const TABLE       = document.querySelector('table');
-const GRADE_TABLE = new GradeTable(TABLE);
+class App {
```

Comment:
Is this supposed to be in main.js?

Reply:
I honestly have no clue where that is coming from my git status is not showing anything is says everything is clean and my main.js file doesn't have that in there
![code](https://user-images.githubusercontent.com/28369586/81482121-93205a80-91e9-11ea-9e12-5e9a6d664259.png)

Reply:
I think you might have copy and pasted your app.js file over your main.js file